tokio-console/src/state/mod.rs

```diff
@@ -5,6 +5,7 @@ use crate::{
     warnings::Linter,
 };
 use console_api as proto;
+use console_api::instrument::Temporality;
```

Comment:
For all other items used from the `console_api` crate, we reference by the "full" path (but starting with `proto`), I think we should do the same here to not make `Temporality` different.

```suggestion
```